### PR TITLE
docs: discourage from caching browsers

### PR DESCRIPTION
**Pull request by mxschmitt**
As per https://github.com/epicweb-dev/epic-stack/pull/204 and https://twitter.com/kentcdodds/status/1670857818604474368.

### CODE REVIEW

File: docs/src/ci.md [hunk 1]
```diff
@@ -799,6 +799,7 @@ executors:
 Note: When using the docker agent definition, you are specifying the resource class of where playwright runs to the 'medium' tier [here](https://circleci.com/docs/configuration-reference?#docker-execution-environment). The default behavior of Playwright is to set the number of workers to the detected core count (2 in the case of the medium tier). Overriding the number of workers to greater than this number will cause unnecessary timeouts and failures.
 
 #### Sharding in CircleCI
+* langs: js
```
Comment (mxschmitt) [inline docs/src/ci.md:802]: drive-by.

**Review by aslushnikov — APPROVED** [review]
